packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1752,7 +1752,7 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
 
     const getSignature = (eventType: Event) => {
       if ([Event.Birth, Event.Death].includes(eventType)) {
-        if (!offlineCountryConfiguration.config.INFORMANT_SIGNATURE) {
+        if (!offlineCountryConfiguration.config.FEATURES.INFORMANT_SIGNATURE) {
```

Comment:
What's the difference between referring through offlineCountryConfiguration and `window.config`?

Reply:
Good catch @rikukissa I think there is no difference. But in my opinion, it should use `window.config` as a single source of truth of application config

Reply:
I agree 👍 easier to search and replace in the future